src/System.Threading.Tasks.Dataflow/tests/Dataflow/EtwTests.cs

```diff
@@ -9,6 +9,7 @@ namespace System.Threading.Tasks.Dataflow.Tests
     public class EtwTests
     {
         [Fact]
+        [ActiveIssue(3520, PlatformID.OSX)]
```

Comment:
Can you verify that the right project is being built and used for OS X? The .WP8.csproj doesn't include the ETW support and would result in this failure. I'm surprised if this attribute is actually needed.